crates/matrix-sdk/src/encryption/verification/qrcode.rs

```diff
@@ -43,6 +43,11 @@ impl QrVerification {
         self.inner.is_done()
     }
 
+    /// Whether the QrCode was scanned by the other device.
+    pub fn is_scanned(&self) -> bool {
```

Comment:
Why change the name? Let's be consistent when we re-export a method.

Reply:
i changed the name to be consistent with the other methods. Since we are already not fully consistent with methods of the  `BaseQrVerification`